packages/components/src/draggable/index.js

```diff
@@ -9,6 +9,29 @@ const cloneHeightTransformationBreakpoint = 700;
 const clonePadding = 0;
 const bodyClass = 'is-dragging-components-draggable';
 
+/**
+ * @typedef RenderProp
+ * @property {(event: import('react').DragEvent) => void} onDraggableStart `onDragStart` handler.
+ * @property {(event: import('react').DragEvent) => void} onDraggableEnd `onDragEnd` handler.
+ */
+
+/**
+ * @typedef Props
+ * @property {(props: RenderProp) => JSX.Element | null} children Children.
+ * @property {(event: import('react').DragEvent) => void} [onDragStart] Callback when dragging starts.
+ * @property {(event: import('react').DragEvent) => void} [onDragOver] Callback when dragging happens over ???.
+ * @property {(event: import('react').DragEvent) => void} [onDragEnd] Callback when dragging ends.
+ * @property {string} [cloneClassname] Classname for the cloned element.
+ * @property {string} [elementId] ID for the element.
+ * @property {any} [transferData] Transfer data for the drag event.
```

Comment:
Isn't there a `DataTransfer` interface or something we can use here? In any case, this is always an object, no?

Reply:
It gets passed to `JSON.stringify` so I think the correct type is `any` as that function's `value` parameter is typed as `any`.